browser/src/react-redux/App/LoginPage.react.js

```diff
@@ -1,111 +1,128 @@
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import setUser from './User/User.action'
-import axios from 'axios'
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
We don't want to have to manually set this for each file.

I found a stack overflow post that may be helpful:
https://stackoverflow.com/questions/43031126/jsx-not-allowed-in-files-with-extension-js-with-eslint-config-airbnb

We don't really want to delay the login form any further with just warnings. Let's go ahead and remove this; create a new ticket; and create a new PR for these changes.

Reply:
I have this fixed in another PR. Would you like it in here as well?